packages/addons/service/mariadb/source/bin/mariadb.start

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+. /etc/profile
+
+MYSQL_ROOT_PASS=123
+
+ADDON_ID=service.mariadb
+ADDON_DIR="$HOME/.kodi/addons/$ADDON_ID"
+ADDON_HOME="$HOME/.kodi/userdata/addon_data/$ADDON_ID"
```

Comment:
`ADDON_DIR` and `ADDON_HOME` are not needed (yes we are doing it wrong elsewhere too) https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/mediacenter/kodi/profile.d/00-addons.conf#L18-L20

Reply:
But you need to call oe_setup_addon() which I don't. BUt I can add it too.